src/DIPS.Xamarin.UI/Controls/Toast/PositionToMarginConverter.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace DIPS.Xamarin.UI.Controls.Toast
+{
+    /// <summary>
+    ///     Converter to convert the Y position of an element relative to the Main page given in percentage to the Margin of
+    ///     the element.
+    /// </summary>
+    public class PositionToMarginConverter : IMarkupExtension, IValueConverter
```

Comment:
This converter is very hard coded and not very generic. I also don't see any usage of it inside of `DXUI`.

For it to be reusable the consumer should be able:
*  Select what kind of position it should use to calculate size and ration (X / Y)
* Select what kind of properties the converter should respect when calculating size and ration.
* Select what kind of `Thickness` the converter should return. 

I think it's easier to not include it and keep this as an converter in each app repository.

Reply:
yes, this was meant to be internal

Reply:
removed the converter and moved to using device pixels.